tree/ntuple/v7/test/ntuple_raw.cxx

```diff
@@ -47,7 +48,9 @@ TEST(RNTuple, Basics)
    auto wrPt = model->MakeField<float>("pt", 42.0);
 
    {
-      auto ntuple = RNTupleWriter::Recreate(std::move(model), "f", fileGuard.GetPath());
+      RNTupleWriteOptions options;
+      options.SetContainerFormat(ENTupleContainerFormat::kRaw);
+      auto ntuple = RNTupleWriter::Recreate(std::move(model), "f", fileGuard.GetPath(), options);
```

Comment:
Side note: The '"f"' made me to have to lookup the signature of Recreate as I imagine it was relate to the file (name or option) and discover it was the (unusual) name of the RNTuple ...